icd/api/vk_descriptor_set.cpp

```diff
@@ -89,6 +89,25 @@ void DescriptorSet<numPalDevices>::Reassign(
 
 }
 
+// =====================================================================================================================
+// Writes the immutable samplers in the layout to memory.
+template <uint32_t numPalDevices>
+void DescriptorSet<numPalDevices>::WriteImmutableSamplers()
+{
+    for (uint32_t deviceIdx = 0; deviceIdx < numPalDevices; deviceIdx++)
+    {
+        for (uint32_t bindingIndex = 0; bindingIndex < Layout()->Info().count; ++bindingIndex) {
+          const DescriptorSetLayout::BindingInfo& bindingInfo = Layout()->Binding(bindingIndex);
+
+          if (bindingInfo.imm.dwSize == 0) continue;
+
+          uint32_t * pSamplerDesc =(Layout()->Info().imm.pImmutableSamplerData + bindingInfo.imm.dwOffset);
```

Comment:
Extra space before *.  There is an extra set of () - some places we want the unnecessary () and others we don't.

Reply:
Done.